modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/RichTextArea.java

```diff
@@ -422,7 +410,7 @@ public final void setCaretBlinkPeriod(Duration period) {
     }
 
     public final Duration getCaretBlinkPeriod() {
-        return caretBlinkPeriod == null ? Params.DEFAULT_CARET_BLINK_PERIOD : caretBlinkPeriod.get();
+        return caretBlinkPeriod == null ? null : caretBlinkPeriod.get();
```

Comment:
This just makes the API harder to use. Why push the burden to the application?

Reply:
reverted.